src/views/About/index.jsx

```diff
@@ -18,21 +18,23 @@ const About = () => {
   return (
     <Grid container justifyContent="center">
       <Grid item xs={12} lg={8}>
-        <Card className={styles.about}>
+        <Card className={styles.about} sx={{ bgcolor: gordonColors.neutral.cardColor}}>
```

Comment:
Hi @Centerville1 , thanks for working on this! I just wanted to mention, before you get too much further on this work, that you will want to rethink how dark mode interacts with MUI components (such as this Card component).

MUI is a library of React components that we use extensively throughout 360. It has a default theme, which we adjust slightly in theme.ts. However, it will be a LOT less work for you and future developers to make use of the built in MUI dark theme than to try to override the styles of every component each time they're used. 

For more details on how to use MUI's built in dark mode support, you can read the [MUI documentation on dark mode](https://mui.com/material-ui/customization/dark-mode/). Notice especially the `palette.mode` setting.